mlCompared.html

```diff
@@ -628,7 +628,7 @@
 let x: int list = [2]
 type listOfListOfInts =
   int list list
-type ('a, 'b) tup = ('a, 'b)
+type ('a, 'b) tup = ('a \* 'b)
```

Comment:
I think these parenthesis should be omitted (though it won't matter in this case, but would in the case of variant constructors IIRC).